action.go

```diff
@@ -3,7 +3,6 @@ package debos
 import (
 	"bytes"
 	"github.com/go-debos/fakemachine"
-	"log"
```

Comment:
keep me plz

Reply:
If I keep it (yet remove LogStart as per above) the build will fail, since it's unused. Am I missing something?

Reply:
This refers to https://github.com/go-debos/debos/pull/380#discussion_r1060542966

Reply:
I suspect you meant the import should be kept alongside the helper, right?

The linked suggests moving the helper to another file, yet this comment says the import should be kept as-is, which will lead to build failure.

Reply:
yes, lets just resolve this one instead...